test/src/unit-filter-pipeline.cc

```diff
@@ -3993,13 +3993,21 @@ void testing_float_scaling_filter() {
       constants::format_version, t, tile_size, cell_size, dim_num);
 
   std::vector<FloatingType> float_result_vec;
-  double scale = 2.53;
-  double foffset = 0.31589;
+  std::random_device rd_so;
+  std::mt19937 gen_so(rd_so());
+  std::uniform_real_distribution<double> dis_so(
+      std::numeric_limits<IntType>::min(), std::numeric_limits<IntType>::max());
```

Comment:
Clang won't let you implicitly convert from a max int to a double because of the precision loss. This line needs to be changed in order to compile on clang.